### ISSUE TITLE
Please cut a release with ruby 2.5+ support

### ISSUE THREAD

**Issue by jrafanie** [mvz/ripper_ruby_parser#35]
It looks like PR #30 fixes an issue I have with ruby 2.5.0.  With ripper_ruby_parser 1.1.2, I get the error below.  Note, on master of ripper_ruby_parser, this error goes away.  Please release a new version 🙇 🙏  ❤️ .  Thank you for your work on this 👍 

```
Error parsing classes in /Users/joerafaniello/Code/manageiq/app/models/dialog.rb:
SexpTypeError: exp must be a Sexp, was Symbol::bodystmt

F.......

Failures:

  1) MiqServer instance after_destroy callback .destroy_linked_events destroys all events associated with destroyed server
     Failure/Error: parsed = RipperRubyParser::Parser.new.parse(content, filename)

     SexpTypeError:
       exp must be a Sexp, was Symbol::bodystmt
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:221:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `block in map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/blocks.rb:161:in `handle_generic_block'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/blocks.rb:18:in `process_do_block'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/blocks.rb:6:in `process_method_add_block'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `block in map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/blocks.rb:83:in `process_bodystmt'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/methods.rb:82:in `block in method_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/methods.rb:72:in `in_method'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/methods.rb:82:in `method_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/methods.rb:19:in `process_defs'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:125:in `process_comment'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `block in map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:68:in `map_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_handlers/blocks.rb:83:in `process_bodystmt'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:219:in `class_or_module_body'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:65:in `process_class'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:125:in `process_comment'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:49:in `block in process_program'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp.rb:65:in `map'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:49:in `process_program'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:269:in `block (2 levels) in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:324:in `error_handler'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:268:in `block in process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:388:in `in_context'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/sexp_processor-4.10.0/lib/sexp_processor.rb:245:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/sexp_processor.rb:39:in `process'
     # /Users/joerafaniello/.gem/ruby/2.5.0/gems/ripper_ruby_parser-1.1.2/lib/ripper_ruby_parser/parser.rb:21:in `parse'
```

**Comment by mvz** [comment]
Hi @jrafanie, I released 1.2.0 with Ruby 2.5 support.

**Comment by jrafanie** [comment]
Thank you @mvz
